pytrafikverket/__main__.py

```diff
@@ -19,7 +19,7 @@
 GET_FERRY_ROUTE = "get-ferry-route"
 SEARCH_FOR_FERRY_ROUTE = "search-for-ferry-route"
 GET_NEXT_FERRY_STOP = "get-next-ferry-stop"
-
+DATE_TIME_INPUT = "%Y-%m-%dT%H:%M:%S"
```

Comment:
Same question here. Why different than in the other file?

Reply:
As you noticed, there was a change at the API to include timezone and milliseconds, but I wanted to keep the pytrafikverket syntax intact to (try) not to make any breaking changes, therefore I needed to have a difference between how the dates was interpreted.

Reply:
That's fine. Quite annoying that the website says something and when you actually test you get a different result.
Will try to test this within the next days to ensure it's working before it's approved and merged.